etna/models/sarimax.py

```diff
@@ -212,6 +214,126 @@ def get_model(self) -> SARIMAXResultsWrapper:
         """
         return self._fit_results
 
+    @staticmethod
+    def _prepare_components_df(components: np.ndarray, model: SARIMAX) -> pd.DataFrame:
```

Comment:
Don't see tests for this method

Reply:
Here we only add names to components and convert to dataframe, so this functionality is tested in `test_components_names`